File: Iniciante/3303/3303.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
  public static void main(String[] args) throws IOException {
    InputStreamReader stream = new InputStreamReader(System.in);
    BufferedReader reader = new BufferedReader(stream);

    String input = reader.readLine().trim();

    if (input.length() >= 10) {
      System.out.println("palavrao");
    } else {
      System.out.println("palavrinha");
    }
  }
}

File: Iniciante/3303/3303.rs
use std::io;

fn main() {
    let mut input = String::with_capacity(21);

    io::stdin().read_line(&mut input).unwrap();

    let input = input.trim();

    if input.len() >= 10 {
        println!("palavrao");
    } else {
        println!("palavrinha");
    }
}
